focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -102,7 +102,7 @@ end
 
 --[[
   -- method figures out the orientation the photo was shot at by looking at the metadata
-  -- returns 90, 270, or 0 (in trigonometric sense)
+  -- returns the rotation in rad in trigonometric sense
```

Comment:
yes, but the metadata is in degrees, LR reports rotation in degrees and this method is a proxy for those. it is easy enough to convert out. and comparing 90 == 90 I dont have to worry about float round off errors